src/Categories/Morphism.agda

```diff
@@ -116,3 +116,7 @@ module ≅ = IsEquivalence ≅-isEquivalence
   ; _≈_           = _≅_
   ; isEquivalence = ≅-isEquivalence
   }
+
+-- diagrammatic composition of morphisms; the symbol is (U+FF1B), "Fullwidth Semicolon", not semicolon (U+003B)
+_；_ : {A B C : Obj} → (f : A ⇒ B) → (g : B ⇒ C) → (A ⇒ C)
```

Comment:
I am super thorn. In the next release of Agda, there will be "fat semicolon", via `\z;` which is much nicer. So I'm kind of leaning towards waiting, as "full width semicolon" does make the typesetting look kind of ugly. And it's way too easy to mistake for `;` which is a problem for usability.

Reply:
Given this, it's probably a bad idea.